_posts/2017-08-16-match-my-route-introduction.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Introduction to MatchMyRoute - an app for matching up experienced with inexperienced cyclists
+meta: Geovation does not only support project of their programme members, but we also work on our own projects. MatchMyRoute
+links into our current annual focus of greener and smarter communities. Its aim is to make cycling more accessible and lower the barrier to picking up cycling. This is done by connecting experienced cyclists who regularly take a route with inexperienced cyclists who can join the experienced on their route.
+author: Max Stephan
+comments: true
+permalink: matchmyroute
+---
+
+![MatchMyRoute Logo](/assets/matchmyroute_logo.png)
+
+# Introduction
+
+This post introduces a project at Geovation on lowering the barrier for people to pick up cycling. The idea is to create a platform for buddying up experienced with inexperienced cyclists so that they can cycle together. We are creating an app for achieving this task which will soon be available for iOS and Android.
```

Comment:
This could get to the point quicker -- the important things are that we have built this app to help people pick up cycling, and that it will be launching soon. I say mention those first, and put how it works and that it was built by Geovation after. And 'this post introduces...' is waffley